src/index.js

```diff
@@ -5,8 +5,10 @@ import Calculator from './pages/calculator';
 import NavLink from './components/nav';
 import './modules-css/calculator.css';
 import './modules-css/nav.css';
-import Home from './pages/Home';
-import Quote from './pages/Quote';
+// eslint-disable-next-line import/extensions
+import Home from './pages/home';
+// eslint-disable-next-line import/extensions
+import Quote from './pages/quote';
```

Comment:
- [ ] Kindly refrain from using `eslint disable`, as that defies the purpose of using linters in the first place.